loopdb/store.go

```diff
@@ -482,24 +495,30 @@ func (s *boltSwapStore) updateLoop(bucketKey []byte, hash lntypes.Hash,
 		if swapBucket == nil {
 			return errors.New("swap not found")
 		}
-		updateBucket := swapBucket.Bucket(updatesBucketKey)
-		if updateBucket == nil {
+		updatesBucket := swapBucket.Bucket(updatesBucketKey)
+		if updatesBucket == nil {
 			return errors.New("udpate bucket not found")
 		}
 
 		// Each update for this swap will get a new monotonically
 		// increasing ID number that we'll obtain now.
-		id, err := updateBucket.NextSequence()
+		id, err := updatesBucket.NextSequence()
 		if err != nil {
 			return err
 		}
 
+		updateBucket, err := updatesBucket.CreateBucket(itob(id))
```

Comment:
nit: just to make it easier to read, I'd call this `nextUpdateBucket`.

Reply:
fixed